app/scripts/modules/core/utils/renderIfFeature.component.ts

```diff
@@ -0,0 +1,23 @@
+import {module} from 'angular';
+
+class RenderIfFeatureController implements ng.IComponentController {
+  public feature: string;
+  public featureEnabled = false;
+
+  constructor(private settings: any) {}
+
+  public $onInit(): void {
+    this.featureEnabled = this.feature && this.settings.feature[this.feature];
+  }
+}
+
+class RenderIfFeatureComponent implements ng.IComponentOptions {
+  public bindings: any = {feature: '@'};
+  public controller: any = RenderIfFeatureController;
+  public transclude = true;
+  public template = `<ng-transclude ng-if="$ctrl.featureEnabled"></ng-transclude>`;
```

Comment:
I had no idea you could use `ng-transclude` as an element.  Where does it place the contents of `<render-if-feature></render-if-feature>`?